kolibri/core/assets/src/vue/icon-button.vue

```diff
@@ -72,6 +72,10 @@
     padding-right: 8px
     height: 36px
 
+  @media screen and (max-width: $portrait-breakpoint)
+    .icon-button
+      margin: 5px
+
```

Comment:
also note that this button is re-used in many places, so adding the style here potentially breaks those. the style should really be moved out